perun-openapi/openapi.yml

```diff
@@ -5829,6 +5980,279 @@ paths:
   #                                               #
   #################################################
 
+  /json/registrarManager/consolidateIdentityUsingToken:
+    get:
+      tags:
+        - RegistrarManager
+      operationId: consolidateIdentityUsingToken
+      summary: Joins current user identity with the one previously provided and referenced by the token.
+      parameters:
+        - { name: token, schema: { type: string }, in: query, required: true, description: "token to be used for joining identities" }
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfUserExtSourcesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/registrarManager/sendInvitation:
+    post:
+      tags:
+        - RegistrarManager
+      operationId: sendInvitation
+      summary: Sends invitation email to user which is not member of VO
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: InputSendInvitation
+              description: "input to sendInvitation"
+              type: object
+              required:
+                - vo
+                - email
+                - language
+              properties:
+                vo: { type: integer, description: "vo id" }
+                name: { type: string, description: "name of person used in invitation email (optional)" }
+                email: { type: string, description: "email address to send invitation to" }
+                language: { type: string, description: "preferred language to use" }
+
+  /json/registrarManager/sendInvitation/g:
+    post:
+      tags:
+        - RegistrarManager
+      operationId: sendInvitationForGroup
+      summary: Sends invitation email to user which is not member of VO and Group
+      description: |
+        Invitation link targets VO application form fist, after submission, Group application form is displayed.
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: InputSendInvitationForGroup
+              description: "input to sendInvitationForGroup"
+              type: object
+              required:
+                - vo
+                - group
+                - email
+                - language
+              properties:
```

Comment:
This method has only primitive parameters, please change the endpoint prefix to /urlinjsonout/ and replace "requestBody" with "parameters". (2)